src/components/TableCellWrapper.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Box } from '@material-ui/core';
+import PropTypes from 'prop-types';
+
+export const TableCellWrapper = (props) => {
```

Comment:
- Add a type prop to decide what heights and widths the Box should be.
- Declare a constant to map types to heights and widths.

Reply:
Basic design implemented